.github/helm-values.yml

```diff
@@ -4,6 +4,8 @@ base_url_protocol: http
 env:
   - name: GRAFANA_CLOUD_NOTIFICATIONS_ENABLED
     value: "False"
+  - name: FEATURE_PROMETHEUS_EXPORTER_ENABLED
+    value: "True"
```

Comment:
I assume prometheus is required for e2e tests

Reply:
Correct